Assets/Plugins/Source/Core/RuntimeConfig.cs

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2024 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+//#define EOS_RUNTIME_NEW_CONFIG_SYSTEM
+
+namespace PlayEveryWare.EpicOnlineServices
+{
+    using Epic.OnlineServices.Auth;
+    using Epic.OnlineServices.IntegratedPlatform;
+    using Epic.OnlineServices.Platform;
+    using System;
+
+#if EOS_RUNTIME_NEW_CONFIG_SYSTEM
+    /// <summary>
+    /// Contains the information necessary to configure the EOS SDK to connect
+    /// properly to Epic Online Services. Is ignorant of platform, because it
+    /// _only_ represents the _current_ runtime values to use. The values of the
+    /// field members defined within will need to be determined at runtime.
+    ///
+    /// _Project Settings_ is the area where there will be options to define
```

Comment:
not pr crit: Your use of markdown here makes me think there's a place that this can be read in markdown format. Is there?

Reply:
Not as far as I know, but it was the best way that I could think of to make use of explicit emphasis.